src/webhook_service/save_webhook/lambda_function.py

```diff
@@ -43,8 +77,9 @@ def lambda_handler(event, context):
             "webhook_name": data.get("webhook_name")
         }
 
+        # Save the item to the main table
         logger.info("Attempting to put item into DynamoDB: %s", item)
-        response = table.put_item(Item=item)
+        response = main_table.put_item(Item=item)
```

Comment:
Based on the [coding guidelines](https://www.notion.so/aws-educate-tw/TPET-Backend-Coding-Guidelines-1596bfee681780b2b309fdc3a2111e26?pvs=4#1596bfee6817808c9ad6e2fea3842e32), you're supposed to move the database interaction into a file named `{table_name}_repository.py`.  

For this case, you should create a `webhook_repository.py` file and define the logic in a class called `WebhookRepository`.

Reply:
The database interaction code has been extracted to [webhook_repository.py](https://github.com/aws-educate-tw/aws-educate-tpet-backend/blob/SCRUM-157-list-webhooks/src/webhook_service/save_webhook/webhook_repository.py)

And the code in [lamda_function.py](https://github.com/aws-educate-tw/aws-educate-tpet-backend/blob/SCRUM-157-list-webhooks/src/webhook_service/save_webhook/lambda_function.py) has been updated also.
```
        # Save the item to the main table
        logger.info("Attempting to put item into DynamoDB: %s", item)
        response = WebhookRepository.save_data(item)
        logger.info("DynamoDB put_item response: %s", response)
```